test/0_ownable.js

```diff
@@ -26,126 +26,125 @@ const constructors = {
       { from: owner }
     ),
   NavCalculator: (owner, dataFeed) => allArtifacts.NavCalculator.new(dataFeed, { from: owner }),
-  InvestorActions: owner => allArtifacts.InvestorActions.new({ from: owner }),
+  InvestorActions: owner => allArtifacts.InvestorActions.new({ from: owner })
 };
 
-contract('OwnableModified', function (accounts) {
-
-  let owner0, owner1, owned;
+contract('OwnableModified', (accounts) => {
+  let owner0;
+  let owner1;
+  let owner2;
+  let owner3;
+  let notOwner0;
+  let notOwnerAddress0;
+  let notOwnerAddress1;
+  let notOwnerAddress2;
+  let notOwnerAddress3;
+  let owned;
   const addressZero = '0x0000000000000000000000000000000000000000';
 
-  before('should prepare', function () {
+  before('should prepare', () => {
     assert.isAtLeast(accounts.length, 2);
-    owner0 = accounts[0];
-    owner1 = accounts[1];
-    owner2 = accounts[2];
-    owner3 = accounts[3];
-    notOwner0 = accounts[4];
-    notOwnerAddress0 = accounts[5];
-    notOwnerAddress1 = accounts[6];
-    notOwnerAddress2 = accounts[7];
-    notOwnerAddress3 = accounts[8];
+    [owner0, owner1, owner2, owner3, notOwner0, notOwnerAddress0, notOwnerAddress1, notOwnerAddress2, notOwnerAddress3] = accounts;
   });
 
-  Object.keys(constructors).forEach(name => {
-
-    describe(name, function () {
-
-      beforeEach('should deploy a new ' + name, function () {
-        return constructors[name](owner0, notOwnerAddress0, notOwnerAddress1, notOwnerAddress2)
+  Object.keys(constructors).forEach((name) => {
+    describe(name, () => {
+      beforeEach(`should deploy a new ${name}`, () => {
+        constructors[name](owner0, notOwnerAddress0, notOwnerAddress1, notOwnerAddress2)
```

Comment:
If there are curly brackets after an arrow function, then you will need to use `return`:
```
() => contructors.... // OK
() => { constructors... } // not OK
() => { return constructors...} // OK
```
For an explanation, see here [es6 arrow functions / return](https://stackoverflow.com/questions/28889450/when-should-i-use-return-in-es6-arrow-functions)

Reply:
Check this throughout the tests since you removed returns in many places